give.php

```diff
@@ -404,10 +404,18 @@ public function display_old_recurring_compatibility_notice()
         }
     }
 
+    /**
+     * Install Give
+     *
+     * Runs on plugin activation and performs initial setup.
+     *
+     * @unreleased set network_wide parameter to true, enabling installing in WP multisite
+     * @since 1.0.0
+     */
     public function install()
     {
         $this->loadServiceProviders();
-        give_install();
+        give_install(true);
```

Comment:
@pauloiankoski Why even have the parameter if it's going to be fixed as true? I can see the function is already checking for `is_multisite()`, so I'm not sure what more value the parameter provides.

Reply:
That function is called in other places like when creating a new subsite. So, even tho, it's a multisite, we don't want to reinstall Give across all subsites, but only in the new one.

![CleanShot 2023-09-28 at 18 40 31](https://github.com/impress-org/givewp/assets/3921017/f92ad178-4e38-4101-8fc9-b341232ca3f3)

Reply:
Ahhh, I see. Good catch. 👍